test/runner/lib/util.py

```diff
@@ -78,6 +78,42 @@
 MODE_DIRECTORY = MODE_READ | stat.S_IWUSR | stat.S_IXUSR | stat.S_IXGRP | stat.S_IXOTH
 MODE_DIRECTORY_WRITE = MODE_DIRECTORY | stat.S_IWGRP | stat.S_IWOTH
 
+ENCODING = 'utf-8'
+
+Text = type(u'')
+
+
+def to_optional_bytes(value, errors='strict'):  # type: (t.Optional[t.AnyStr], str) -> t.Optional[bytes]
```

Comment:
Is it worthwhile taking a copy of the text utils we already have in Ansible. They have a `passthru` option to define things like what to do when a None object is being encoded.

Reply:
The optional versions exist so that the PEP 484 type hints are accurate. The optional versions may return `None`, while the non-optional versions do not.